packages/grafana-toolkit/bin/grafana-toolkit.js

```diff
@@ -5,26 +5,46 @@ const path = require('path');
 
 let includeInternalScripts = false;
 
+const isLinkedMode = () => {
+  let pwd = process.env['PWD'];
+
+  // In circleci we are in linked mode. Detect by using the circle working directory,
+  // rather than the present working directory.
+  if (process.env['CIRCLE_WORKING_DIRECTORY']) {
+    pwd = process.env['CIRCLE_WORKING_DIRECTORY'];
+  }
+
+  if (path.basename(pwd) === 'grafana-toolkit') {
+    return true;
+  }
+
+  const linkedDir = `${pwd}/node_modules/@grafana/toolkit`.replace('~', process.env.HOME);
+  if (fs.existsSync(linkedDir)) {
+    const tkStat = fs.lstatSync(linkedDir);
+    if (tkStat.isSymbolicLink()) {
+      return true;
+    }
+  }
+
+  return false;
```

Comment:
This is simpler and avoids a race condition (file could be deleted after `existsSync` and before `lstatSync`):
```js
try {
  return fs.lstatSync(`${pwd}/node_modules/@grafana/toolkit`.replace('~', process.env.HOME)).isSymbolicLink();
} catch {
  return false;
}
```